source/extensions/filters/http/ext_proc/ext_proc.h

```diff
@@ -358,14 +376,19 @@ class FilterConfig {
   const std::vector<envoy::extensions::filters::http::ext_proc::v3::ProcessingMode>
       allowed_override_modes_;
   const ExpressionManager expression_manager_;
-
   const ImmediateMutationChecker immediate_mutation_checker_;
 
   const std::function<std::unique_ptr<OnProcessingResponse>()> on_processing_response_factory_cb_;
 
   ThreadLocal::SlotPtr thread_local_stream_manager_slot_;
   const std::chrono::milliseconds remote_close_timeout_;
   const Http::Code status_on_error_;
+
+  const absl::optional<const envoy::config::core::v3::TypedExtensionConfig>
+      processing_request_modifier_config_;
+
+  Extensions::Filters::Common::Expr::BuilderInstanceSharedConstPtr builder_;
+  Server::Configuration::CommonFactoryContext& common_factory_context_;
```

Comment:
Will these be needed if the PRM (processing request modifier) is created upon instantiation of the ext-proc filter instance?